python/dask_cudf/dask_cudf/io/orc.py

```diff
@@ -146,6 +165,15 @@ def to_orc(
     storage_options: None or dict
         Further parameters to pass to the bytes backend.
     compression : string or dict, optional
+    stripe_size_bytes: integer or None, default None
+        Maximum size of each stripe of the output.
+        If None, 67108864 (64MB) will be used.
+    stripe_size_rows: integer or None, default None 1000000
+        Maximum number of rows of each stripe of the output.
+        If None, 1000000 will be used.
+    row_index_stride: integer or None, default None 10000
```

Comment:
Nitpicky, but feel like we can remove the implicit default value as long as it is mentioned in the param description:

```suggestion
    stripe_size_rows: integer or None, default None
        Maximum number of rows of each stripe of the output.
        If None, 1000000 will be used.
    row_index_stride: integer or None, default None
```

Reply:
Done, thanks